src/Summoner/Template.hs

```diff
@@ -109,66 +109,96 @@ createStackTemplate ProjectData{..} = Dir (toString repo) $
+            exposed-modules:    $libModuleName
+                                $preludeMod
+            $createWarnings
+            build-depends:      $base
+                              $customPreludePack
+            default-language:   Haskell2010
+            $defaultExtensions
         $endLine
         |]
 
     createCabalExe :: Text -> Text
     createCabalExe r =
         [text|
         executable $repo
-          hs-source-dirs:      app
-          main-is:             Main.hs
-          ghc-options:         -Wall -threaded -rtsopts -with-rtsopts=-N
-          build-depends:       $base
-                             $r
-                             $customPreludePack
-          default-language:    Haskell2010
-          $defaultExtensions
+            hs-source-dirs:     app
+            main-is:            Main.hs
+            $createWarnings
+            build-depends:      $base
+                              $r
+                              $customPreludePack
+            default-language:   Haskell2010
+            $defaultExtensions
         $endLine
         |]
 
     createCabalTest :: Text
     createCabalTest =
         [text|
         test-suite ${repo}-test
-          type:                exitcode-stdio-1.0
-          hs-source-dirs:      test
-          main-is:             Spec.hs
-          build-depends:       $base
-                             , $repo
-                             $customPreludePack
-          ghc-options:         -Wall -Werror -threaded -rtsopts -with-rtsopts=-N
-          default-language:    Haskell2010
-          $defaultExtensions
+            type:                exitcode-stdio-1.0
+            hs-source-dirs:      test
+            main-is:             Spec.hs
+            build-depends:       $base
+                               , $repo
+                               $customPreludePack
+            $createWarnings
+            default-language:   Haskell2010
+            $defaultExtensions
         $endLine
         |]
 
     createCabalBenchmark :: Text -> Text
     createCabalBenchmark r =
         [text|
         benchmark ${repo}-benchmark
-          type:                exitcode-stdio-1.0
-          default-language:    Haskell2010
-          ghc-options:         -Wall -Werror -O2 -threaded -rtsopts -with-rtsopts=-N
-          hs-source-dirs:      benchmark
-          main-is:             Main.hs
-          build-depends:       $base
-                             , gauge
-                             $customPreludePack
-                             $r
-          $defaultExtensions
+            type:               exitcode-stdio-1.0
+            default-language:   Haskell2010
+            $createWarnings
+            hs-source-dirs:     benchmark
+            main-is:            Main.hs
+            build-depends:      $base
+                              , gauge
+                              $customPreludePack
+                              $r
```

Comment:
I just noticed that `benchmarks` and `executable` has `r` variable. So the code can be improved even further. Could you, please, do the following?

1. Swap lines with `$customPreludePack` and `$r`
2. Add `r` variable to `createCabalTest` function?